pkg/core/imageprocessing/operation.go

```diff
@@ -0,0 +1,16 @@
+package imageprocessing
```

Comment:
I think this package should belongs to the asset gear, not the core library.

Reply:
Just in case anyone else wants to have image processing.